source/js/buyers-guide/template-js-handler/pni-category-dropdown.js

```diff
@@ -22,7 +22,7 @@ export default () => {
 
   function resizeCategoryNavigation() {
     const categoryLinks = [
-      ...document.querySelectorAll("#pni-category-wrapper > .multipage-link"),
+      ...document.querySelectorAll("#pni-category-wrapper .multipage-link"),
```

Comment:
What do you think about using that new id here instead of the `pni-category-wrapper`?

Reply:
sure lets do that